.flowconfig

```diff
@@ -15,6 +15,7 @@
 
 [untyped]
 .*/node_modules/trezor-connect/lib/popup/PopupManager.js
+.*/node_modules/trezor-connect/lib/iframe/builder.js
```

Comment:
Explanation: both PopupManager and now builder.js throw an error on Flow. Until Trezor upgrades their version of Flow and fix the related issue, we have to suppress it on our side